### ISSUE TITLE
Feature idea: add Gmail Actions to notifications

### ISSUE THREAD

**Issue by goldenapples** [rmccue/Falcon#66]
It seems to me that Gmail's Actions functionality would make new post/comment notifications even better by allowing you to jump directly to the post from your inbox, without having to open the email, scroll to the bottom, and click the "View it on {blogname}" link.

From what I see, this is as simple to implement as adding some schema data to a script tag in the message:

https://developers.google.com/gmail/markup/actions/declaring-actions

This is how it looks in the case of GitHub notifications:
![image](https://user-images.githubusercontent.com/665992/129800141-6ef30e1a-5449-437b-b9de-34b88fc4c40f.png)

A similar action that just said "View post" and linked to the original post would be nice.

**User story:** Anecdotally, many people read posts in their email but don't click through to comment or react. Being able to open a post directly from one's inbox list would remove a barrier to engagement.

**Comment by rmccue** [comment]
I actually built this feature out (check the message source for H2 emails), but unfortunately, Gmail only adds domains for the feature which send large volumes of traffic. We don’t qualify, unfortunately, so have to close as wontfix.

**Comment by Nikschavan** [comment]
I got this working for Falcon emails in my company. I just had to register with google to get the sender email address whitelisted - https://developers.google.com/gmail/markup/registering-with-google

On the above link they do mention that they require volume of at least hundred emails per day, which we don't comply. I don't think they checked the volume to be honest. They just verified SPF, DKIM and validated the schema markup and the email was whitelisted in a couple of weeks.

Also, the schema markup did not work until #67 was updated.
